src/hb-ot-color-colr-table.hh

```diff
@@ -150,9 +618,18 @@ struct COLR
   bool sanitize (hb_sanitize_context_t *c) const
   {
     TRACE_SANITIZE (this);
-    return_trace (likely (c->check_struct (this) &&
-			  (this+baseGlyphsZ).sanitize (c, numBaseGlyphs) &&
-			  (this+layersZ).sanitize (c, numLayers)));
+    if (unlikely (!(c->check_struct (this) &&
```

Comment:
The `check_struct` here is not very nice. We should also check for version 0.
Copy the sanitize style from `GDEF` table.